index.html

```diff
@@ -417,18 +418,19 @@ <h1></h1>
         </div>
     </div>
     <div class="footer_v" id="footer">
-        <div>
-            <a href="#" onclick="hide()">hide</a>
-            <a href="#" onclick="popsearch()">search</a>
-            <a href="#" onclick="toggletextsize()">toggle text size</a>
-        </div>
-        <div>
-            <p id="loginstr" class="admin none">logged in as <span id="username">guest</span></p>
-            <a href="#" onclick="showpwchange()" id="pwa" class="admin none">change password</a>
-            <a href="#" onclick="logout()" id="logout" class="admin none">logout</a>
-            <a href="#" onclick="showlogin()" id="logina" class="admin">edit</a>
-            <a href="#" onclick="showabout()">about</a>
-        </div>
+        <nav class="ftr_sub" id="ftr_gen">
+            <img id="banner" src="./images/banner.png">
+            <div class="ftr_btn"><a href="#" class="btn_text" onclick="hide()">hide</a></div>
+            <div class="ftr_btn"><a href="#" class="btn_text" onclick="popsearch()">search</a></div>
+            <div class="ftr_btn"><a href="#" class="btn_text" onclick="toggletextsize()">text size</a></div>
+        </nav>
+        <nav class="ftr_sub" id="ftr_usr">
+            <div id="loginstr" class="admin none">logged in as <span id="username">guest</span></div>

```

Comment:
this occupies the same space as a button.. so it could technically have the same class as the other buttons w/o interfering with any other rules? :D

Reply:
this is entirely possible, i couldn't really test it because my local environment doesn't correctly have login working. i kind of wanted the entire navbar section to have a different class so that it could be colored differently (as per the concept) but i wasn't sure how to do that without a proper test environment with all the interactibles working. i was gonna leave the work of making and assigning those classes to you 
![test](https://github.com/user-attachments/assets/f02be849-90f6-490a-a27a-15e33e63f4ca)